classes/install/Upgrade.inc.php

```diff
@@ -309,7 +304,7 @@ public function localizeIssueCoverImages()
                 break;
             default: fatalError('Unknown database type!');
         }
-        $issueDao->flushCache();
+        $deprecatedIssueDao->flushCache();
```

Comment:
The upgrade scripts generally avoid calling on DAO functions (because these will evolve with new code and break older upgrade steps), so we often worked around this by instantiating any old DAO then calling `::update` et al with SQL coded directly. You can adapt this to Laravel tools without problems (and without having to rewrite everything using a query builder) by using `DB::select`, `DB::update`, `DB::delete` instead of `DAO::retrieve` and `DAO::update`.